Testscripts/Windows/ETHTOOL-GRO-LRO.ps1

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+param([String] $TestParams,
+      [object] $AllVmData)
+
+function Main {
+    Write-LogInfo "Generating constants.sh ..."
+    $constantsFile = "$LogDir\constants.sh"
+    Set-Content -Value "TEST_PLATFORM=$($TestPlatform)" -Path $constantsFile
+    Write-LogInfo "constants.sh created successfully..."
+
+    Copy-RemoteFiles -uploadTo $allVmData.PublicIP -port $allVmData.SSHPort `
+        -files "$constantsFile" -username $user -password $password -upload
+    #
+    # Run the guest VM side script
+    #
+    try {
+        Run-LinuxCmd -username $user -password $password -ip $allVmData.PublicIP -port $allVmData.SSHPort `
+            "bash ETHTOOL-GRO-LRO.sh" -runAsSudo | Out-Null
+
+        $status = Run-LinuxCmd -ip $allVMData.PublicIP -port $allVMData.SSHPort `
+            -username $user -password $password -command "cat state.txt"
+        Copy-RemoteFiles -downloadFrom $allVMData.PublicIP -port $allVMData.SSHPort `
+            -username $user -password $password -download `
+            -downloadTo $LogDir -files "*.txt, *.log"
+        if ($status -imatch "TestFailed") {
+            Write-LogErr "Test failed."
```

Comment:
throw

Reply:
Maybe remove the line 47 with the initial value. And here just set fail or abort value with throw statement.

Reply:
Thank you @juhlee-microsoft ! We are still discussing about the possibility of detecting platform inside VM, such as by checking the route "169.254.169.254", or checking wala. IMO there's a risk to pass "TestPlatform" from outside because I just learned that there was another platform named "Ready". What's your opinion about that? Thanks!